src/NumericHelper.php

```diff
@@ -83,4 +83,21 @@ public static function isInteger($value): bool
     {
         return filter_var($value, FILTER_VALIDATE_INT) !== false;
     }
+
+    /**
+     * Checks whether the given string represents a positive integer number.
+     *
+     * @param string $string
+     *
+     * @return bool
+     */
+    public static function isPositiveInteger(string $string): bool
```

Comment:
Check only strings is not consistent to the `isInteger` method with mixed argument.

Maybe also use mixed argument and goal of this method will be to check that type casting value to integer will give positive integer?

Reply:
`+1` string will be cast to positive integrer.